pegasus_wrapper/scripts/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Dockerfile, Image, Container
+FROM python:3.6
+
+LABEL name="Pegasus Wrapper Demo Container"
+LABEL version=0
+LABEL revision=1
+LABEL maintainer="jacobl@isi.edu"
+
+WORKDIR /home/app
+COPY add_y.py /home/app/
+COPY sort_nums_in_file.py /home/app/
+
+CMD echo "Docker running"
```

Comment:
`CMD` is the default command that this image will execute, so by default it will complete the `echo` and then terminate.  So the message could also be "Docker running/completed."